magstock/__init__.py

```diff
@@ -57,7 +65,7 @@ def food_cost(self):
 
     @property
     def auto_food(self):
-        return self.badge_type in [c.STAFF_BADGE, c.GUEST_BADGE]
+        return self.badge_type in [c.STAFF_BADGE, c.GUEST_BADGE] or hasattr(self, "auto_food_extra") and self.auto_food_extra
```

Comment:
@kitsuta the reason is that the magstock plugin probably shouldn't know anything about bands.  so this is it setting up a hook where other plugins can say which things should get free food.  In that sense, the band plugin is modifying the magstock plugin, and the magstock plugin doesn't have to know anything about bands in order to work.

Either way is probably Ok since magstock will probably use the band plugin forever from now on, I just figured it would be good to maintain a one way interface between these two plugins where bands plugin can mess with magstock stuff, but magstock plugin doesn't have to know anything about bands